Makefile

```
@@ -121,7 +121,7 @@ run-docker:
 
 .PHONY: nginx
 nginx: python
-	@tox -qe dev --run-command 'docker-compose run --rm --service-ports nginx-proxy'
+	@tox -qe dev --run-command 'docker compose run --rm --service-ports nginx-proxy'
```

```suggestion
	@docker compose run --rm --service-ports nginx-proxy
```

I don't think this needs tox anymore